### PR TITLE
Encapsulate the game in a React component

### PR DESCRIPTION
**Pull request by jgiovaresco**
The ultimate goal would be deleting language, explain, victory Phaser states and replace them with React components

### CODE REVIEW

File: src/game/CycleGame.ts [hunk 1]
```diff
@@ -13,8 +13,8 @@ import { Languages } from './states/languages';
 export class CycleGame {
     private game: Phaser.Game;
 
-    constructor() {
-        this.game = new Phaser.Game(BASE_SIZE, BASE_SIZE, Phaser.CANVAS, 'cycleCanvas');
+    constructor(private readonly htmlElement: string, public readonly baseSize = BASE_SIZE) {
```
Comment (gaetanmaisse) [inline src/game/CycleGame.ts]: `public readonly baseSize = BASE_SIZE` why not `private`?

File: src/app/App.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                </p>
+                <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
+                    Learn React
+                </a>
+            </header>
+        </div>
+    );
+};
+
+export default App;
```
Comment (gaetanmaisse) [inline src/app/App.tsx]: I think we must avoid `default` exports, see a lot of good reasons here: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

File: src/game/CycleGame.ts (hunk 1 above)
Comment (jgiovaresco) [reply src/game/CycleGame.ts]: It needs to be public to be able to set the height and width of the HTML element:
https://github.com/cycle-game/Cycle/pull/39/files#diff-34456421648850188ad56fcd6df47b2bR21

**Pull request merged**